static-location.js

```diff
@@ -1,16 +1,20 @@
+import { currentPathname } from "./utils.js";
+
 // Generates static `useLocation` hook. The hook always
 // responds with initial path provided.
 // You can use this for server-side rendering.
-export default (path = "/", { record = false } = {}) => {
```

Comment:
Unfortunately, this won't work since `base` is passed to the hook itself, not the factory. We have recently published support for parent routers in #265, and it assumes that base is always sent to the hook.

Reply:
What won’t work?

Reply:
Or maybe we have different views on how this is going to work. In my understanding, the purpose of base path in static location is to produce the same output when you swap your browser hook with static one for Server-Side Rendering. 

Consider this app:
```js
import useBrowserLocation from "wouter/use-location";

<Router hook={useBrowserLocation} base="/app">
  <Route path="/users">
    <Link to="/home" /> {/* real path is "/app/home" *}
   </Route>
</Router>
```

And then let's say you'd like to swap your location hook with a static one to be able to render a specific page from the server:

```js
import staticLocation from "wouter/static-location";

// Important: `staticLocation` is not a hook just yet, it is a factory to produce the desired static hook
const useStaticLocation = staticLocation("/users")

<Router hook={useStaticLocation} base="/app">
  <Route path="/users">
    <Link to="/home" /> {/* in the markup the href is "/app/users" *}
   </Route>
</Router>
```

Reply:
In wouter the consideration about location hooks is that they are functions that may or may not accept an object with options and must return a pair of location and location setter.

`type LocationHook = (opts?: {}) => [string, (newLocation: string) => void]`

Wouter always sends `base` when it calls provided `useLocation``

Reply:
How is it now?